packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/ViewDiagramDescriptionConverter.java

```diff
@@ -732,4 +742,43 @@ private String evaluateString(AQLInterpreter interpreter, VariableManager variab
     private IDiagramContext getDiagramContext(VariableManager variableManager) {
         return variableManager.get(IDiagramContext.DIAGRAM_CONTEXT, IDiagramContext.class).orElse(null);
     }
+
+    private Function<VariableManager, List<Action>> convertActions(org.eclipse.sirius.components.view.diagram.NodeDescription viewNodeDescription, AQLInterpreter interpreter, ViewDiagramDescriptionConverterContext converterContext) {
```

Comment:
You don't need to convert them once again

Reply:
Why once again? If this is the second time I convert the actions, where is the first time. In _ViewActionsProvider_?